r2i/iengine.h

```diff
@@ -62,6 +64,32 @@ class IEngine {
       in_frame,
       r2i::RuntimeError &error) = 0;
 
+  /**
+   * \brief Set an instance of IPreprocessing into Engine.
+   * \param preprocessing Shared pointer on an IPreprocessing object.
```

Comment:
`on` -> `of`

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/584edecea9ba1fd3e0b6dfca877ed4d0b94247c0)